hw_diag/tests/diagnostics/test_env_var_diagnostics.py

```diff
@@ -56,5 +56,7 @@ def test_env_vars_success(self):
             'FIRMWARE_VERSION': 'foo',
             'FW': 'foo',
             'VARIANT': 'foo',
-            'VA': 'foo'
+            'VA': 'foo',
+            'FIRMWARE_SHORT_HASH': 'foo',
+            'FSH': 'foo'
```

Comment:
No need for the shorter `FSH` name. The short keys are only 2-characters long and we are moving away from them gradually. So instead of `VA` we try to use `VARIANT`. These legacy 2-character keys will be removed once all code using them has been updated to use the more friendlier keys.